test/frontend/components/branchViewLink.test.js

```diff
@@ -8,44 +8,58 @@ describe('<BranchViewLink/>', () => {
   const testSite = {
     defaultBranch: 'default-branch',
     demoBranch: 'demo-branch',
-    viewLink: 'https://prod-url.com',
-    demoViewLink: 'https://demo-url.com',
+    domain: 'https://prod-url.com',
+    demoDomain: 'https://demo-url.com',
     owner: 'test-owner',
     repository: 'test-repo',
     awsBucketName: 'test-bucket',
     s3ServiceName: 'federalist-production-s3',
   };
 
+  const getViewLink = (build) => {
```

Comment:
The UI is just spitting out  whatever the backend is giving it, does it care what it actually is? Do the tests that are using this actually care? They look like they are just checking that the `href` matches `viewLink` so let's just test that, I don't think we need to logic here to generate the correct url.

Reply:
are you suggesting bringing the Build model into the frontend?

Reply:
No, i'm saying all we're testing here is that the `href` reflects the value passed in by the `viewLink` prop, so it can be any random string.

Reply:
So we don't need a function to generate a "real" url.